testing/integration/validate_items_deprecated_list_test.go

```diff
@@ -0,0 +1,59 @@
+package integration
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/elastic/elastic-agent/pkg/testing/define"
+	"github.com/stretchr/testify/require"
+)
+
+type testKibanaApiKey struct {
+	ID       string `json:"id"`
+	Name     string `json:"name"`
+	Active   bool   `json:"active"`
+	PolicyID string `json:"policy_id"`
+	APIKey   string `json:"api_key"`
+}
+
+type deprecatedBody struct {
+	List []testKibanaApiKey `json:"list"`
+}
+
+type newBody struct {
+	Items []testKibanaApiKey `json:"items"`
+}
+
+// TODO: Remove test after list deprecation is complete
```

Comment:
I would like to add an issue link to remove the link from the kibana api, if anyone knows of such an issue please comment here.

Reply:
let's crete an issue in our repo. this change is breaking and will be removed in 9.0 
we can create a PR targeting main with skip backport right away after this one is merge, removing this test

Reply:
Created this issue https://github.com/orgs/elastic/projects/979/views/23?pane=issue&itemId=92232925&issue=elastic%7Celastic-agent%7C6451